Cargo.toml

```diff
@@ -19,14 +19,14 @@ nightly_docs = []
 version = "0.1"
 optional = true
 
-
 [dev-dependencies]
 rand = "0.7"
 criterion = "0.3"
+anyhow = "1.0.32"
```

Comment:
I'm curios, why don't simply "1.0"?
(Asking because it's what they suggest in the docs, but maybe there is a reason to specify the minor)

Reply:
Oh, no reason specifically, I just took the latest one. I can switch it back to 1.0 if you want.

Reply:
I'll switch to it since if that's what they suggest we should probably do it.